shared/src/main/scala/squants/mass/AreaDensity.scala

```diff
@@ -54,7 +54,7 @@ object KilogramsPerHectare extends AreaDensityUnit with UnitConverter {
   val conversionFactor = 1/(100*100d)
 }
 
-object GramsPerSquareCentimeter extends AreaDensityUnit with UnitConverter {
+object GramsPerSquareCentimeter extends AreaDensityUnit with UnitConverter with SiUnit {
```

Comment:
Are you derived units too? I'm not sure if `AreaDensity` has a formal SI definition, Do you have a reference?